packages/teams-js/test/public/runtime.spec.ts

```diff
@@ -282,6 +282,7 @@ describe('runtime', () => {
     return `${version}.1`;
   }
 
+  /* eslint-disable-next-line recommend-const-enums/recommend-const-enums */
```

Comment:
> /* eslint-disable-next-line recommend-const-enums/recommend-const-enums */ [](http://example.com/codeflow?start=1&length=75)

Can you say more about why this one needs to be suppressed?

Reply:
It doesn't. It's a leftover from when I was developing the plugin. Since the test folder doesn't impact the bundle size, I forgot to make them `const` too. I went ahead and made them `const`